include/MySQL_Monitor.hpp

```diff
@@ -428,6 +489,13 @@ class MySQL_Monitor {
 	void evaluate_aws_aurora_results(unsigned int wHG, unsigned int rHG, AWS_Aurora_status_entry **lasts_ase, unsigned int ase_idx, unsigned int max_latency_ms, unsigned int add_lag_ms, unsigned int min_lag_ms, unsigned int lag_num_checks);
 	unsigned int estimate_lag(char* server_id, AWS_Aurora_status_entry** ase, unsigned int idx, unsigned int add_lag_ms, unsigned int min_lag_ms, unsigned int lag_num_checks);
 //	void gdb_dump___monitor_mysql_server_aws_aurora_log(char *hostname);
+
+private:
+	void monitor_ping_async(SQLite3DB* monitordb, SQLite3_result* resultset);
```

Comment:
The new responsibilities delegated into this set of functions, from their original single threaded from are not obvious. Would be great to have here some doc about what is expected from them, and what they assume. For instance, they assume the variables to be updated by the calling thread, but they perform themselves the updating of certain data on the supplied `monitordb`. Because they perform certain operations in the same way as the old set of functions, is a good opportunity to document both by adding this new doc. And to clarify the stages of all the monitoring threads:

1. Connection obtention and operations delegation (different in this new version)
2. Configuration gathering for performing the required fetched operations.
3. Fetch operations themselves.
4. Monitoring actions using fetch data (including handling of errors over the fetched data).
5. Final cleanup.